util/common/metrics.go

```diff
@@ -130,6 +198,71 @@ func SendCollectDMetrics(metricPerInterval int, sendingInterval, duration time.D
 		}
 	}
 
+}
+func processFile(filePath string, startTime int64) error {
+	data, err := os.ReadFile(filePath)
+	if err != nil {
+		fmt.Println("Error reading file:", err)
+		return nil
+	}
+
+	//replace START_TIME with the current time
+	modifiedData := strings.ReplaceAll(string(data), "START_TIME", fmt.Sprintf("%d", startTime))
+
+	//curl command
+	url := "http://127.0.0.1:4316/v1/metrics"
```

Comment:
nit: might want to make the port a variable for later use if we want to change it